validation-test/compiler_crashers_2_fixed/1f6ae695bebb359b.swift

```diff
@@ -1,3 +1,3 @@
 // {"signature":"makeBinOp(swift::ASTContext&, swift::Expr*, swift::Expr*, swift::Expr*, swift::PrecedenceGroupDecl*, bool)"}
-// RUN: not --crash %target-swift-frontend -typecheck %s
+// RUN: not %target-swift-frontend -typecheck %s
```

Comment:
Turns out this also affects pattern type-checking, I'll look into that in a follow-up